LongWarOfTheChosen/Config/XComClassData.ini

```diff
@@ -1613,3 +1613,246 @@ bHasClassMovie=true
+							 (AbilityType=(AbilityName="SmartMacrophages", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon)), \\
+							 (RandomDeckName="SkirmisherTier2XComAbilities")), \\
+			  aStatProgression=((StatType=eStat_Offense,StatAmount=2), (StatType=eStat_HP,StatAmount=1), (StatType=eStat_Strength,StatAmount=1), (StatType=eStat_Hacking,StatAmount=0), (StatType=eStat_CombatSims,StatAmount=0)))
+
+; captain
++SoldierRanks=(AbilitySlots=((AbilityType=(AbilityName="Formidable", ApplyToWeaponSlot=eInvSlot_Unknown)), \\
+							(AbilityType=(AbilityName="Retribution", ApplyToWeaponSlot=eInvSlot_SecondaryWeapon)), \\
+							(AbilityType=(AbilityName="FullThrottle", ApplyToWeaponSlot=eInvSlot_Unknown)), \\
+							(RandomDeckName="SkirmisherTier2XComAbilities")), \\
+			  aStatProgression=((StatType=eStat_Offense,StatAmount=2), (StatType=eStat_HP,StatAmount=0), (StatType=eStat_Strength,StatAmount=0), (StatType=eStat_Hacking,StatAmount=5), (StatType=eStat_CombatSims,StatAmount=0)))
+
+; major 
++SoldierRanks=(AbilitySlots=((AbilityType=(AbilityName="Judgment", ApplyToWeaponSlot=eInvSlot_Unknown)), \\
+							(AbilityType=(AbilityName="RunandGun")), \\
+							(AbilityType=(AbilityName="Aggression")), \\
+							(RandomDeckName="SkirmisherTier3XComAbilities")), \\
+			  aStatProgression=((StatType=eStat_Offense,StatAmount=1), (StatType=eStat_HP,StatAmount=1), (StatType=eStat_Strength,StatAmount=0), (StatType=eStat_Hacking,StatAmount=0), (StatType=eStat_CombatSims,StatAmount=0)))
+
+; colonel
++SoldierRanks=(AbilitySlots=((AbilityType=(AbilityName="ManualOverride")), \\
+							(AbilityType=(AbilityName="Implacable")), \\
+							(AbilityType=(AbilityName="BringEmOn")), \\
+							(RandomDeckName="SkirmisherTier3XComAbilities")), \\
+			  aStatProgression=((StatType=eStat_Offense,StatAmount=1), (StatType=eStat_HP,StatAmount=1), (StatType=eStat_Strength,StatAmount=1), (StatType=eStat_Hacking,StatAmount=5), (StatType=eStat_CombatSims,StatAmount=0)))
+
+
+; brigadier  
++SoldierRanks=(AbilitySlots=((AbilityType=(AbilityName="Battlelord")), \\
+							(AbilityType=(AbilityName="LW2WotC_Reckoning", ApplyToWeaponSlot=eInvSlot_SecondaryWeapon)), \\
+							(AbilityType=(AbilityName="TacticalSense", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon)), \\
+							(RandomDeckName="SkirmisherTier3XComAbilities")), \\
+			  aStatProgression=((StatType=eStat_Offense,StatAmount=2), (StatType=eStat_HP,StatAmount=0), (StatType=eStat_Strength,StatAmount=0), (StatType=eStat_Hacking,StatAmount=5), (StatType=eStat_CombatSims,StatAmount=0)))
+
+
+
+
+[Reaper X2SoldierClassTemplate]
+!RandomAbilityDecks=()
++RandomAbilityDecks=(DeckName="ReaperXComAbilities", \\
+					Abilities=((AbilityName="Shredder",  ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="Squadsight"), \\
+							   (AbilityName="HoloTargeting",  ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   (AbilityName="Kubikuri", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
```

Comment:
I'm 50/50 on this, mainly only because it is at least better than the rng based repeater execute that it used to be. With kubikiri you actually have to invest into crit for it to work. Though I guess it could be made more interesting as well.

Reply:
I'm actually going to keep this here because I think a lot has to be dumped into the reaper to be able to get crit high enough to be able to use this properly. Might be something we can look into in the future if we think it is out of hand.